tsconfig.base.json

```diff
@@ -10,122 +10,63 @@
     "importHelpers": true,
     "target": "es2015",
     "module": "esnext",
-    "lib": [
```

Comment:
Are we aligned on formatting?  My settings have a different line break.

Reply:
I'm aware of this issue where the formatting of this files and other JSON files may go back and forth between two different formattings. I think that I have a ticket that I will resurface now that we are more to use the monorepo.

Reply:
@sagely1 This issue is now tracked in #2786